pkg/authorization/reaper/cluster_role.go

```diff
@@ -53,7 +53,7 @@ func (r *ClusterRoleReaper) Stop(namespace, name string, timeout time.Duration,
 		}
 	}
 
-	if err := r.roleClient.ClusterRoles().Delete(name, &metav1.DeleteOptions{}); err != nil && !kerrors.IsNotFound(err) {
+	if err := r.roleClient.ClusterRoles().Delete(name, &metav1.DeleteOptions{}); err != nil {
```

Comment:
`&& !kerrors.IsNotFound(err)` looks to have been an intentional choice. Also, we risk breaking existing scripts by changing behavior for this case.
Tagging @simo5 and @deads2k to confirm it'd be fine to remove that check.

Reply:
I do not see a huge issue, broken scripts can be fixed